user-journey-tests/helpers/testHelpers/movementLicense.js

```diff
@@ -0,0 +1,23 @@
+import landingPage from '../../page-objects/landingPage.js'
+import toFromFarmPage from '../../page-objects/toFromFarmPage.js'
+import parishHoldingNumberPage from '../../page-objects/parishHoldingNumberPage.js'
+import newAddressPage from '../../page-objects/newAddressPage.js'
+
+const lineOne = '37 Made up lane'
+const townOrCity = 'Gotham'
+const postcode = 'SW1A2AA'
+
+const completeMovementLicenceAnswers = async () => {
```

Comment:
This `origin/summary` page is just one of 4 task list section pages, rather than the full movement licence 'check' page.

worth changing the wording here, if the purpose of this is to get up to the point of `origin/summary`?